alert-city/client
# Feat: Internationalization-backend-debug

## TL;DR
- Added two different style icon into public img folder
- Added redirection.ts
- Added switchTheme.ts
- Added a dialog when the account not activated
- Adjusted the NavigationBarLayout.tsx
- Added reminder when  hover on the stay signed in
- Optimized the redirection logic in the middleware.ts
- Optimized the logic and UI for profile update page
- Added icons for profile page navigation and setting list

## How Has This Been Tested?
1. Login
2. Switch the theme and language in the preference page see effect
3. Hover your mouse on the stay signed to see the effect
4. Register a new account but don't activate it, and then login to see the effect

## Screenshots
<img width="1920" alt="image" src="https://github.com/user-attachments/assets/726fee1f-4c5d-4021-9189-ccca87e330db">

<img width="1920" alt="image" src="https://github.com/user-attachments/assets/1627c6e7-dd00-40f5-8c69-11fb161e074e">

<img width="1920" alt="image" src="https://github.com/user-attachments/assets/1361de38-fb00-4e53-bc68-d25bed5481d6">

## Resolve 
Resolve CP-25
